app/src/pages/p/[projectSlug]/settings/index.tsx

```diff
@@ -180,6 +181,19 @@ export default function Settings() {
             </VStack>
             <OpenaiApiKeyDisplay />
             <Divider />
+            <VStack alignItems="flex-start">
+              <Subtitle>Rate Limit</Subtitle>
+              <HStack>
+                <Text color="gray.500">Concurrent requests:</Text>
+                <Text fontWeight="bold">{selectedProject?.rateLimit}</Text>
+                <InfoCircle tooltipText="The limits are based by the number of concurrent requests. There is no limit on the total number of requests you can send, but rather on how many you can send at the same time." />
```

Comment:
Instead of putting this in an info box let's just put the text inline similar to the text under "OpenAI API Key".

Text if you haven't added a payment method:
"Your rate limit represents the number of concurrent requests you can send to our servers without getting a 429 "Too Many Requests" error. [Add a payment method](link to add payment method page) to increase your rate limit to 20 concurrent requests automatically."

Text if you have a payment method:
"Your rate limit represents the number of concurrent requests you can send to our servers without getting a 429 "Too Many Requests" error. To increase your rate limit, email [support@openipe.ai](support@openpipe.ai)."

Reply:
Current projects without a payment method already have a rate limit of 1000. I guess in this place, I still need to hardcode a check if the current rate limit is 3.